.github/workflows/cli-runner-test.yml

```
@@ -0,0 +1,160 @@
+name: Slack CLI Command Runner Tests 
+
+on:
+  pull_request:
+
+jobs:
+  test-all:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

```suggestion
      - name: Checkout the current code
        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

👁️‍🗨️ suggestion: Guards against unexpected changes upstream!